content/influxdb/cloud-dedicated/reference/client-libraries/v2/python.md

```diff
@@ -0,0 +1,193 @@
+---
+title: Python client library
+seotitle: Use the InfluxDB Python client library
+list_title: Python
+description: >
+  Use the InfluxDB Python client library to interact with InfluxDB.
+menu:
+  influxdb_cloud_dedicated:
+    name: Python
+    parent: v2 client libraries
+influxdb/cloud-dedicated/tags: [client libraries, python]
+aliases:
+  - /influxdb/cloud-dedicated/reference/api/client-libraries/python/
+  - /influxdb/cloud-dedicated/reference/api/client-libraries/python-cl-guide/
+  - /influxdb/cloud-dedicated/tools/client-libraries/python/
+weight: 201
+---
+
+Use the [InfluxDB Python client library](https://github.com/influxdata/influxdb-client-python) to integrate InfluxDB into Python scripts and applications.
+
+This guide presumes some familiarity with Python and InfluxDB.
+If just getting started, see [Get started with InfluxDB](/influxdb/cloud-dedicated/get-started/).
+
+## Before you begin
+
+1. Install the InfluxDB Python library:
+
+    ```sh
+    pip install influxdb-client
+    ```
+
+2. Ensure that InfluxDB is running.
+   If running InfluxDB locally, visit http://localhost:8086.
+   (If using InfluxDB Cloud, visit the URL of your InfluxDB Cloud UI.
+   For example: https://us-west-2-1.aws.cloud2.influxdata.com.)
+
+## Write data to InfluxDB with Python
+
+We are going to write some data in [line protocol](/influxdb/cloud-dedicated/reference/syntax/line-protocol/) using the Python library.
+
+1. In your Python program, import the InfluxDB client library and use it to write data to InfluxDB.
+
+   ```python
+   import influxdb_client
+   from influxdb_client.client.write_api import SYNCHRONOUS
+   ```
+
+2. Define a few variables with the name of your [bucket](/influxdb/cloud-dedicated/organizations/buckets/), [organization](/influxdb/cloud-dedicated/organizations/), and [token](/influxdb/cloud-dedicated/security/tokens/).
+
+   ```python
+   bucket = "<my-bucket>"
+   org = "<my-org>"
+   token = "<my-token>"
+   # Store the URL of your InfluxDB instance
+   url="http://localhost:8086"
+   ```
```

Comment:
~~~suggestion
2. Define a few variables with the name of your [database](/influxdb/cloud-dedicated/admin/databases/) (bucket), organization (required, but ignored), and [token](/influxdb/cloud-dedicated/admin/tokens/).

   ```python
   bucket = "<my-database>"
   org = "ignored"
   token = "<my-token>"
   # Store the URL of your InfluxDB instance
   url="https://cluster-id.influxdb.io"
   ```
~~~